src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SetAsideActions.java

```diff
@@ -12,18 +12,18 @@
 @Getter
 @AllArgsConstructor
 public enum SetAsideActions implements CcdCallbackMap {
-    GRANT("grant","Grant Set Aside Application", SET_ASIDE_GRANTED, "Set aside application granted", "Set aside application granted", null),
-    REFUSE("refuse","Refuse Set Aside Application", SET_ASIDE_REFUSED, "Set aside application refused", "Set aside application refused", DwpState.SET_ASIDE_REFUSED),
-    ISSUE_DIRECTIONS("issueDirections","IssueDirections", SET_ASIDE_ISSUE_DIRECTIONS, "Set aside application issue directions", "Set aside application issue directions", null),
-    REFUSE_SOR("issueDirections","IssueDirections", SET_ASIDE_REFUSED_SOR, "Set aside application refused & request statement of reasons", "Set aside application refused & request statement of reasons", null);
+    GRANT("grant","Grant Set Aside Application", SET_ASIDE_GRANTED, "Set aside application granted", "Set aside application granted", null, null),
+    REFUSE("refuse","Refuse Set Aside Application", SET_ASIDE_REFUSED, "Set aside application refused", "Set aside application refused", DwpState.SET_ASIDE_REFUSED, null),
+    ISSUE_DIRECTIONS("issueDirections","IssueDirections", SET_ASIDE_ISSUE_DIRECTIONS, "Set aside application issue directions", "Set aside application issue directions", null, null),
+    REFUSE_SOR("issueDirections","IssueDirections", SET_ASIDE_REFUSED_SOR, "Set aside application refused & request statement of reasons", "Set aside application refused & request statement of reasons", DwpState.SET_ASIDE_REFUSED, InterlocReviewState.REVIEW_BY_JUDGE);
```

Comment:
Is this an existing mistake? REFUSE_SOR has the same ccdDefinition and descriptionEn as ISSUE_DIRECTIONS

Reply:
Seems like an exisiting mistake, changed it to match refuse